### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by tonyantony300**
Right container and cards almost done

### CODE REVIEW

File: src/lib/components/dashboard/credentials/CredentialCard.svelte [hunk 1]
```diff
@@ -1,69 +1,100 @@
...
+  import More from '../../basic/more.svelte';
+  import Locked from '../../basic/locked.svelte';
+  import Eye from '../../basic/eye.svelte';
+  import Unlocked from '../../basic/unlocked.svelte';
+  import SensitiveEye from '../../basic/sensitiveEye.svelte';
+  import SensitiveEye2 from '../../basic/sensitiveEyeBlue.svelte';
     const dispatch = createEventDispatcher();
 
     export let credential;
+    export let visibility = writable(false)
```
Comment (AbrahamGeorge8547) [inline src/lib/components/dashboard/credentials/CredentialCard.svelte]: why is visiblity a store var?

Comment (AbrahamGeorge8547) [reply src/lib/components/dashboard/credentials/CredentialCard.svelte]: export is usually used when its a prop from parent component, refer how credentialList calls credentialCard.

**Review by AbrahamGeorge8547 — APPROVED** [review]

File: src/lib/components/dashboard/credentials/CredentialCard.svelte [hunk 2]
```diff
@@ -44,10 +45,15 @@
     }
 
     function makeVisible(){
-        visibility.set(true);
-        setTimeout(()=> {
-            visibility.set(false)
-        }, 3000)
+        if(visibility){
+            visibility = false;
```
Comment (AbrahamGeorge8547) [inline src/lib/components/dashboard/credentials/CredentialCard.svelte]: just use !visibility

Comment (tonyantony300) [reply src/lib/components/dashboard/credentials/CredentialCard.svelte]: Done check, merge ready

**Pull request merged**